<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tools::FspPrinterDialogPrecorded</class>
 <widget class="QWidget" name="Tools::FspPrinterDialogPrecorded">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
